### PR TITLE
changed colors of HTML property and value

### PR DESCRIPTION
**Pull request by laminko**

### CODE REVIEW

**Comment by v3ss0n** [comment]
@lminko  I am not planning to add more base color. Look at other colors and add there.
also make a branch , modiify and make a pull request , that way easier to maintain.

Currently this change effects all other colors , if you want to change only html specific , you need to scope them to their selectors.

File: styles/colors.less [hunk 1]
```diff
@@ -39,9 +39,12 @@
 @base08: #f2777a;
 @base09: #f99157;
 @base0A: #ffcc66;
-@base0F: #d27b53;
-@base0E: #ccaac8;
-@base0D: #6699cc;
-@base0C: #66cccc;
 @base0B: rgb(18, 255, 169);
-
+@base0C: #66cccc;
```
Comment (v3ss0n) [inline styles/colors.less:43]: Not going to add those new Base color, i will only use 16 base colors , no more than that base 16 + sytanx base colors.

File: styles/base.less [hunk 2]
```diff
@@ -112,7 +112,7 @@ atom-text-editor.is-focused, :host {
 }
 
 .storage {
-  color: @base08;
+  color: @base14;
```
Comment (v3ss0n) [inline styles/base.less:115]: Can you explain me why you change it? I am not adding this change , this change every storage colors.

File: styles/base.less [hunk 3]
```diff
@@ -189,7 +189,7 @@ atom-text-editor.is-focused, :host {
   }
 
   &.other.attribute-name {
-		color: @base0E;
+		color: @base11;
```
Comment (v3ss0n) [inline styles/base.less:192]: correct selector is : 

```
 &.other.attribute-name.html {
```

File: styles/base.less (hunk 2 above)
Comment (laminko) [inline styles/base.less:115]: Yes, i thought it looks better clear for me.

File: styles/base.less (hunk 3 above)
Comment (laminko) [inline styles/base.less:192]: thank you, bro. you are right.

**Comment by v3ss0n** [comment]
I added HTML Entity name support in the update, for your own colors you can modify and release as package too.

File: styles/colors.less (hunk 1 above)
Comment (laminko) [inline styles/colors.less:43]: ah, I see, bro. No more additional colors. Thank you.

**Pull request closed without merging**